src/components/beacon/beacon.test.tsx

```diff
@@ -22,9 +22,24 @@ describe('EuiBeacon', () => {
     expect(component).toMatchSnapshot();
   });
 
-  test('accepts size', () => {
-    const component = render(<EuiBeacon size={14} {...requiredProps} />);
+  describe('props', () => {
+    describe('color', () => {
+      COLORS.forEach((color) => {
+        it(`${color} is rendered`, () => {
+          const component = render(
+            <EuiBeacon color={color} {...requiredProps} />
+          );
 
-    expect(component).toMatchSnapshot();
+          expect(component).toMatchSnapshot();
+        });
+      });
+    });
+    describe('size', () => {
+      it('accepts size', () => {
+        const component = render(<EuiBeacon size={14} {...requiredProps} />);
+
+        expect(component).toMatchSnapshot();
+      });
+    });
```

Comment:
Very minor / optional code organization - I'd add a newline between blocks and leave this as its own `test` vs giving it a describe block
```suggestion

    test('size', () => {
      const component = render(<EuiBeacon size={14} {...requiredProps} />);

      expect(component).toMatchSnapshot();
    });
```

Reply:
@constancecchen I agree. I tried to commit, but build process is failing with this error.

```
src/components/modal/modal.tsx:84:60 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(props: EuiFocusTrapProps | Readonly<EuiFocusTrapProps>): EuiFocusTrap', gave the following error.
    Type '{ children: Element; initialFocus: string | HTMLElement | (() => HTMLElement) | undefined; scrollLock: true; preventScrollOnFocus: true; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<EuiFocusTrap> & Pick<Readonly<EuiFocusTrapProps> & Readonly<...>, "className" | ... 17 more ... | "closeOnMouseup"> & Partial<...> & Partial<...>'.
      Property 'preventScrollOnFocus' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<EuiFocusTrap> & Pick<Readonly<EuiFocusTrapProps> & Readonly<...>, "className" | ... 17 more ... | "closeOnMouseup"> & Partial<...> & Partial<...>'.
  Overload 2 of 2, '(props: EuiFocusTrapProps, context: any): EuiFocusTrap', gave the following error.
    Type '{ children: Element; initialFocus: string | HTMLElement | (() => HTMLElement) | undefined; scrollLock: true; preventScrollOnFocus: true; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<EuiFocusTrap> & Pick<Readonly<EuiFocusTrapProps> & Readonly<...>, "className" | ... 17 more ... | "closeOnMouseup"> & Partial<...> & Partial<...>'.
      Property 'preventScrollOnFocus' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<EuiFocusTrap> & Pick<Readonly<EuiFocusTrapProps> & Readonly<...>, "className" | ... 17 more ... | "closeOnMouseup"> & Partial<...> & Partial<...>'.

84       <EuiFocusTrap initialFocus={initialFocus} scrollLock preventScrollOnFocus>
                                                                                                      ~~~~~~~~~~~~~~~~~~~~



Found 1 error.
```

![image](https://user-images.githubusercontent.com/72073614/203615966-de4b721a-83d2-41ad-888d-d504d409e5d9.png)


I checked `EuiFocusTrap`'s props and there's no `preventScrollOnFocus` prop.

Reply:
That's a super bizarre error since it's not related to EuiBeacon 🤔 - I recently merged in latest main into this branch in order to kickstart CI, so I'm guessing what happened is you may need to re-run `yarn` to fix type definitions.

No worries at all though as this was a super optional comment! Thank you again for your great work!